### DISCUSSION TITLE
Meeting Notes auto generating

### DISCUSSION THREAD

**Discussion by LACSupport** [code4recovery/12-step-meeting-list#647] [Q&A] [answered]
Hello, I am having a problem in which meeting notes are being automatically entered for each meeting but I am not writing anything in on some of them, I don't want the text to appear on every single meeting. How do I disable this?

This is what it looks like when I am editing.
![Editing_Meeting](https://user-images.githubusercontent.com/88159477/150002008-c1e73aa6-b0fd-49fa-b02b-99a8eafbe0c0.JPG)

This is what shows up, the circled area is the section  ant removed.
![Meeting_Preview](https://user-images.githubusercontent.com/88159477/150002006-45c74f5a-6fa3-44c9-afc2-5a879a1cc153.JPG)

**Comment by Grateful-Viv** [comment, marked as answer]
If it’s not an open AA meeting, then uncheck open as a type.

**Reply by kiyote33** [reply]
Hi @LACSupport,

Expanding on @Grateful-Viv's answer, the circled text is not part of the meeting notes, but is the type description for either the Open type (as in this case) or the Closed type, which comes immediately after the Types listing and before the Notes listing. You might be able to prevent these text injections by placing an override in the functions.php file of your child theme. Something like **$tsml_programs[$tsml_program]['type_descriptions'] = null;** might do the trick...

I actually tested the following code in my sandbox which prevented the type descriptions from displaying:
**$tsml_programs[$tsml_program]['type_descriptions']['O'] = '';
$tsml_programs[$tsml_program]['type_descriptions']['C'] = '';**
